egs/ljspeech/tts1/run.sh

```diff
@@ -42,6 +43,12 @@ model=model.loss.best
 n_average=1 # if > 0, the model averaged with n_average ckpts will be used instead of model.loss.best
 griffin_lim_iters=64  # the number of iterations of Griffin-Lim
 
+# objective evaluation related
+asr_model="librispeech.transformer.ngpu4"
+eval_tts_model=1                               # 1:evaluate tts model, 0:evaluate ground truth
+voc="GL"                                       # the selection of vocoder
+api="v2"                                       # v1, v2
```

Comment:
Do v1 and v2 both work? How does this make differences? Is there any benefit to expose the option?

Reply:
Yes, both v1 and v2 work. The difference between v1 and v2 is a little in CER results (approximately 0.1 more or less).
I used v1 for generating att_ws but that function removed to pass checks.

I removed this option `api="v2"` and set `--api v2` by referring `recog_wav.sh`.